.gitignore

```diff
@@ -2,4 +2,5 @@
 *.pyc
 __pycache__
 assets
-target
\ No newline at end of file
+target
+.idea
```

Comment:
As this is IDE based, and every user has a different IDE, I prefer this to be in each users local computer as a global gitignore: https://blog.soltysiak.it/en/2017/12/how-to-ignore-files-only-on-my-machine/

Reply:
Ooo that's cool, I did not know you could do that :eyes: